chart/compass/templates/portieris-service-monitor.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if eq .Values.global.metrics.enabled true -}}
+{{ if eq .Values.global.portieris.isEnabled true }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: portieris
+  namespace: portieris
+  labels:
+    prometheus: monitoring
+    app: portieris
+spec:
+  endpoints:
+    - port: metrics
+      scheme: https
+      tlsConfig:
+        ca:
+          secret:
+            key: ca.crt
+            name: portieris-certs
+        cert:
+          secret:
+            key: tls.crt
+            name: portieris-certs
+        keySecret:
+          key: tls.key
+          name: portieris-certs
+        insecureSkipVerify: true
+      metricRelabelings:
+        - sourceLabels: [ __name__ ]
+          regex: ^(portieris_pod_admission_decision_deny_count)$
```

Comment:
Looking at the portieris documentation, there's also a `portieris_pod_admission_decision_allow_count` metric is there any reason it isn't exposed?

Reply:
Because this metric doesn't give us info that could be a "warning".